packages/faust-nx/src/getWordPressProps.tsx

```diff
@@ -62,6 +66,9 @@ export async function getWordPressProps(options: GetWordPressPropsConfig) {
     resolvedUrl,
   }) as DocumentNode;
 
+
+  // setup auth client
```

Comment:
We may want to denote this as a "todo" otherwise it looks like that's what we are doing.